client/src/components/RecommendationsList.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import Listing from './Listing.jsx';
+
+// make list component that pulls each object from the listings array
+// and passes each one down to the listing component (coming soon)
+const RecommendationsList = (props) => {
+  // each item is a recommendation object
+  // SYNTAX NOTE: no curly brackets after map callback function
+  var items = props.listings.map((listing, index) =>
+    <Listing key={index} listing={listing} />
```

Comment:
Is there another way you can uniquely reference the key rather than using the index? Perhaps a listing id number?

Reply:
Right. I didn't think about that. I'll definitely change that.